quarkchain/cluster/grpc_client.py

```diff
@@ -13,10 +13,28 @@ def __init__(self, host: str = HOST, port: int = PORT):
         channel = grpc.insecure_channel("{}:{}".format(host, str(port)))
         self.client = grpc_pb2_grpc.ClusterSlaveStub(channel)
 
-    def set_rootchain_confirmed_block(self) -> bool:
-        request = grpc_pb2.SetRootChainConfirmedBlockRequest()
+    def add_root_block(self, libra_root_block) -> bool:
```

Comment:
1. instead of removing the previous method, how about adding this as a new method?
2. the argument should probably be a normal root block ?

Reply:
Should we send a simplified libra root block here? Or do you mean we can just define the libra block structure here without define it in core.py?

Reply:
Done.